### DISCUSSION TITLE
Daily 14/09

### DISCUSSION THREAD

**Discussion by BraianTroncoso** [grupo-ispc-2023-2-0/workflix-app#57] [Daily]
Con respecto al proyecto:

- ¿Qué hice ayer?
- ¿Qué hice o haré hoy?
- Impedimentos o dificultades

**Comment by BraianTroncoso** [comment]
Hoy sigo con android, ahora estoy estudiando Java. También voy a realizar algun cuestionario de ciberseguridad.
Si me sobra tiempo durante el dia, quiero ver si puedo hacer un crud y la conexión esta vez con mysql(MVP).

**Comment by MarielaGomez** [comment]
Hoy subí el diagrama UML de los casos de usos, sus funcionamientos, actores y roles. Continúo con ampliar marco teórico de Testing y practicar Java

**Comment by NataliaAlvarezIspc** [comment]
Ayer corregí y subí a la wiki las correcciones del diagrama de clases. Tambíen estuve leyendo material de app móviles y de testing. Por la noche participé de la clase de inglés.
Hoy tuve que hacer reposo así que solo participaré de la clase de App por la tarde.
